src/components/v5/common/AvatarUploader/hooks.ts

```diff
@@ -47,6 +47,12 @@ export const useAvatarUploader = ({ updateFn }: UseAvatarUploaderProps) => {
     } catch (e) {
       if (e.message.includes('exceeded the maximum')) {
         setUploadAvatarError(DropzoneErrors.TOO_LARGE);
+      } else if (
+        e.message.includes(
+          "Pica: cannot use getImageData on canvas, make sure fingerprinting protection isn't enabled",
```

Comment:
I'd normally store this as a `enum` or just plain `type`, but since it's a single error message, it feel kinda "weird" to do that.

Reply:
I was debating on whether or not to store it as an enum at first, but I thought I'll just add it as a plain string for now since it's only ever used here and this will likely go away once the proper solution gets applied. BUT if we do end up having another error message to handle, I shall use an enum to make things a bit more tidy! 😄 @rdig